src/Game.java

```diff
@@ -0,0 +1,160 @@
+        this.player2 = player2;
+    }
+
+    public String setWinner(Player name) {
+        return winnerName = name.getName();
+    }
+
+    public String setLoser(Player name) {
+        return loserName = name.getName();
+    }
+
+    public String getWinner() {
+        return winnerName;
+    }
+
+    public String getLoser() {
+        return loserName;
+    }
+
+    public void runGame() {
+        int choice;
+        setBoard();
+        System.out.println("Ok, let's play!");
+        while (!declareLoser(totalSticks)) {
+            choice = currentPlayer.makeChoice();
+            checkChoice(choice);
+            totalSticks -= choice;
+            System.out.format("There are %d sticks remaining.\n", totalSticks);
+            switchPlayer();
+        }
+    }
+
+    public void setBoard() {
+        int stickSelect = 0;
+        while (stickSelect < STICKMIN || stickSelect > STICKMAX) {
+            System.out.println("How many sticks will be on the table (10 - 100): ");
+            stickSelect = Integer.parseInt(scanner.nextLine());
+            if (stickSelect < STICKMIN || stickSelect > STICKMAX) {
+                System.out.println("Nope, enter a number between 1 & 100.");
+            }
+        }
+        totalSticks = stickSelect;
+    }
+
+    public void switchPlayer() {
+        if (currentPlayer == player1) {
+            currentPlayer = player2;
+        } else {
+            currentPlayer = player1;
+        }
+    }
+
+    public boolean declareLoser(int amountRemaining) {
+        boolean gameOver = false;
+        if (amountRemaining == 1) {
+            switchPlayer();
+            setWinner(currentPlayer);
+            System.out.println(currentPlayer.getName() + " is the winner!");
+            switchPlayer();
+            setLoser(currentPlayer);
+            System.out.println(currentPlayer.getName() + " is the loser! :'(");
+            gameOver = true;
+        } else if (amountRemaining == 0) {
+            setWinner(currentPlayer);
+            System.out.println(currentPlayer.getName() + " is the winner!");
+            switchPlayer();
+            setLoser(currentPlayer);
+            System.out.println(currentPlayer.getName() + " is the loser! :'(");
+            gameOver = true;
+        }
+        return gameOver;
+    }
+
+    public int checkChoice(int choice) {
+        while ( choice < SELECTMIN || choice > SELECTMAX || choice > totalSticks) {
+            if (choice > totalSticks) {
+                System.out.format("There are only %d sticks. You cannot take more than that.\n", totalSticks);
+                currentPlayer.makeChoice();
+            } else if (choice < SELECTMIN || choice > SELECTMAX) {
+                System.out.println("You can only take 1, 2 or 3 sticks.");
+                currentPlayer.makeChoice();
+            }
+        }
+        return choice;
+    }
+
+    public Map loadFile(Map players) throws FileNotFoundException {
```

Comment:
`loadFile` and `saveFile` probably do not belong in the `Game` object.  From OOD perspective the game is just 1 object representing 1 game.  These methods keep track of all games won by all players which is a level outside of an individual game.